src/tests/mocks/matchMedia.js

```diff
@@ -0,0 +1,9 @@
+window.matchMedia = jest.fn().mockImplementation(query => {
```

Comment:
for the function body just only have `return` statement, I think we could use
```suggestion
window.matchMedia = jest.fn().mockImplementation(query => ({
matches: false,
    media: query,
    onchange: null,
    addListener: jest.fn(),
    removeListener: jest.fn(),
    }))
```